Firestore/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
   ```
   To go back to using the binary distribution of Firestore, quit Xcode and
   open Xcode like normal, without the environment variable. (#11492)
+- [feature] Added support for having multiple inequality fields in compound queries. (#11626)
```

Comment:
this should be added under Unreleased title. Like this:

```
# Unreleased
- [feature] Added support for having multiple inequality fields in compound queries. (#11626)

# 10.12.0
...
```

here's an example: https://github.com/firebase/firebase-ios-sdk/pull/9233/files#diff-9588906d19a574a4898d5a2b35871ab5bbb1c09688bd076c4692dc3184f5a104R1

During a new release, the person doing the release takes care of adding the new version number and moving the CHANGELOG entries under the new version.

Reply:
Note: removed the changelog.